elide-async/src/main/java/com/yahoo/elide/async/service/DefaultAsyncQueryDAO.java

```diff
@@ -148,7 +139,7 @@ public AsyncQuery updateStatus(String asyncQueryId, QueryStatus status) {
                 }
                 return loaded;
             });
-        } catch (ParseException e) {
+        } catch (Exception e) {
```

Comment:
The DAO should not swallow exceptions.  Utilities need to propagate exceptions and let the calling code handle them (or the utility will have limited reuse and can only be used in limited contexts).